tests.md

```diff
@@ -0,0 +1,238 @@
+# Tests
+
+The WordPress Hosting Team provides tools for hosting companies to run the WordPress automated tests on their infrastructure to improve compatibility with WordPress. These results can be published them on the [Host Test Result information page](https://make.wordpress.org/hosting/test-results/), to help WordPress' compatibility with hosts as well.
+
+It consists of two tools: the Runner, which is part of the core's PHPUnit tests, which work on a hosting, and optionally [sends the information to the results page](https://make.wordpress.org/hosting/test-results/); and the Reporter, which is the plugin that [works on the hosting page](https://make.wordpress.org/hosting/) and shows the results.
+
+## What is it
+
+### Runner
+
+Hosting companies can have hundreds of websites hosted with WordPress, so it's important to make sure their configuration is as compatible as possible with the software.
```

Comment:
I'm not sure about including a number here, since it varies a lot between hosts, from several to millions.
Either way, it's important to users for the environment to be supported.

I don't think this is a blocker for merging it, but it stood out for me, in case you can think of a different wording.